### ISSUE TITLE
Compiling for iOS 16.0, but module 'SkipWeb' has a minimum deployment target of iOS 17.0

### ISSUE THREAD

**Issue by FadelSultan** [skiptools/skip-web#3]
Hello, 
How are you doing?

I use this package and install it by adding a package in Xcode.

`import SkipWeb`

I mate this issue 

> Compiling for iOS 16.0, but module 'SkipWeb' has a minimum deployment target of iOS 17.0

how can I fix?

thank's

**Comment by FadelSultan** [comment]
I update it 
see the images 

![2](https://github.com/user-attachments/assets/cbbdf5b4-babf-4b00-b773-886ab3487bbc)
![3](https://github.com/user-attachments/assets/ee2a7dbb-c1ac-4b2c-8313-32f0a2844a99)
<img width="1171" alt="Screenshot 2024-11-04 at 9 58 30 PM" src="https://github.com/user-attachments/assets/3ac7ddaf-5b4b-479b-8c81-8e967abbead4">
<img width="874" alt="Screenshot 2024-11-04 at 10 01 19 PM" src="https://github.com/user-attachments/assets/f00ecad9-08c1-47f0-bbb0-604ff6dbab52">

**Comment by marcprux** [comment]
I've sometimes found that when I change the minimum iOS target I need to clean the build folder and sometimes even restart Xcode. Can you try that and let us know if it helps?

If you still have problems, please post your complete `Package.swift`.

**Comment by FadelSultan** [comment]
I try and still same issue 

IOS 18.1
Xcode Version 16.1 (16B40)

import PackageDescription

let package = Package(
    name: "skip-web",
    defaultLocalization: "en",
    platforms: [.iOS(.v17), .macOS(.v13), .tvOS(.v17), .watchOS(.v10), .macCatalyst(.v17)],
    products: [
        .library(name: "SkipWeb", targets: ["SkipWeb"]),
    ],
    dependencies: [
        .package(url: "https://source.skip.tools/skip.git", from: "1.0.10"),
        .package(url: "https://source.skip.tools/skip-ui.git", from: "1.6.1")
    ],
    targets: [
        .target(name: "SkipWeb", dependencies: [.product(name: "SkipUI", package: "skip-ui")], resources: [.process("Resources")], plugins: [.plugin(name: "skipstone", package: "skip")]),
        .testTarget(name: "SkipWebTests", dependencies: ["SkipWeb", .product(name: "SkipTest", package: "skip")], resources: [.process("Resources")], plugins: [.plugin(name: "skipstone", package: "skip")]),
    ]
)
<img width="1512" alt="Screenshot 2024-11-05 at 4 28 17 PM" src="https://github.com/user-attachments/assets/da1816dd-c7ba-4dec-8351-ffedf0e8facd">


<img width="818" alt="Screenshot 2024-11-05 at 4 25 51 PM" src="https://github.com/user-attachments/assets/be9c403d-30b3-40f8-b77d-b1ca10c994c0">

**Comment by FadelSultan** [comment]
Do you mean this?
if yes how can I edit it? 
because I can't edit in normal 
<img width="1512" alt="Screenshot 2024-11-05 at 7 21 24 PM" src="https://github.com/user-attachments/assets/60458a7f-6599-456c-a8da-e198ee2a8f6a">

Thank's

**Comment by marcprux** [comment]
No, I mean the Package.swift within your own project's Xcode project folder.

For example, the Net Skip app (which uses skip-web) looks like this:

<img width="1789" alt="Screenshot 2024-11-05 at 11 34 40" src="https://github.com/user-attachments/assets/59e17ae5-2f36-441f-8331-b8ddce807ffa">

**Comment by FadelSultan** [comment]
This is the project 
https://drive.google.com/file/d/1IwENwwP-kQxLPkyPEKw6Ao1ay8q6goLb/view?usp=sharing 

if you want to try :)

this is the issues after do update 
<img width="1512" alt="Screenshot 2024-11-05 at 8 45 04 PM" src="https://github.com/user-attachments/assets/adb27c8f-8378-4b39-9d27-ebbb9f9f255f">

**Comment by marcprux** [comment]
The `Package.swift` in that project does not have a dependency on skip-web.

**Comment by FadelSultan** [comment]
Are there any new things?
Thank's

**Comment by FadelSultan** [comment]
<img width="626" alt="Screenshot 2024-11-10 at 9 12 02 AM" src="https://github.com/user-attachments/assets/12f491fd-e812-4840-9ed6-2f8214725c5d">

I'm not sure if you mean this.

**Comment by marcprux** [comment]
The Package.swift that you posted in the linked Google Drive project looks like this:

```swift
// swift-tools-version: 5.9
// This is a Skip (https://skip.tools) package,
// containing a Swift Package Manager project
// that will use the Skip build plugin to transpile the
// Swift Package, Sources, and Tests into an
// Android Gradle Project with Kotlin sources and JUnit tests.
import PackageDescription

let package = Package(
    name: "amiralnal-project",
    defaultLocalization: "en",
    platforms: [.iOS(.v17), .macOS(.v13), .tvOS(.v16), .watchOS(.v9), .macCatalyst(.v16)],
    products: [
        .library(name: "amiralnahlApp", type: .dynamic, targets: ["amiralnahl"]),
    ],
    dependencies: [
        .package(url: "https://source.skip.tools/skip.git", from: "1.1.18"),
        .package(url: "https://source.skip.tools/skip-ui.git", from: "1.0.0")
    ],
    targets: [
        .target(name: "amiralnahl", dependencies: [.product(name: "SkipUI", package: "skip-ui")], resources: [.process("Resources")], plugins: [.plugin(name: "skipstone", package: "skip")]),
        .testTarget(name: "amiralnahlTests", dependencies: ["amiralnahl", .product(name: "SkipTest", package: "skip")], resources: [.process("Resources")], plugins: [.plugin(name: "skipstone", package: "skip")]),
    ]
)
```

It should have a dependency on `skip-web` (like you posted), and also the target should have a dependency on the `SkipWeb` product.
